src/views/Simulation/components/PixiComponents/components/pixiNode.js

```diff
@@ -1,50 +1,53 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
-import * as PIXI from 'pixi.js';
+import { Graphics, Text, Container } from 'pixi.js';
+import { ensureTransportEdge, removeTransportEdge } from './pixiTransport';
+
+const getNodeName = (id) => {
+  const s = String(id);
+  return s.startsWith('node-') ? s : `node-${s}`;
+};
 
 const STYLE = {
   coreIdleRadius: 1,
   coreFill: 0xb9bac0,
   ringColor: 0xffffff,
-  ringWidth: 2,
+  ringWidth: 0.25,
   fontSize: 18,
   fontColor: 0x000000,
-  minRadius: 2,
-  maxRadius: 20,
+  minRadius: 1,
+  maxRadius: 30,
 };
 
-function drawCore(g, r, fill, ringColor, ringWidth) {
+function drawCore(g, r, fill, labelText) {
```

Comment:
For the future, in view of uniformisation, prefer as way as possible, the arrow form :
`const drawCore() => {`